central/src/lib/API/GameQuestions.ts

```diff
@@ -1,22 +1,37 @@
 import {
   IAPIClients,
   IGameQuestion,
-  CreateGameQuestionsInput
+  CreatePublicGameQuestionsInput,
+  CreatePrivateGameQuestionsInput,
+  PublicPrivateType
 } from '@righton/networking';
 
-export const createGameQuestions = async (apiClients: IAPIClients, createGameQuestionsInput: CreateGameQuestionsInput): Promise<IGameQuestion | null> => {
+export const createGameQuestions = async (
+  type: PublicPrivateType,
```

Comment:
all calls to the backend now include a `PublicPrivateType` that describes if the call is for the Public or Private DynamoDB table.